NSS-Spontaneous-Spoonbills/Sprint3
# Js products home page

Closes #15

```sh
git fetch --all
git checkout js_productsHomePage
code .
django_data.sh website faker_factory
go to   http://127.0.0.1:8000/website
```


## Steps to Test
1. Login or go to Home page
1. There should be a total of 20 products on the home page
1. The product should be clickable and lead to a description page



Outline the steps to test




## Impacted Areas in Application

**New Files**:


**Modified Files**:


**Deleted Files**:



## Mentions @username

Tag users that need to review this code

## Testing


## Deploy Notes

Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
1. Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
